client/my-sites/current-site/notice.jsx

```diff
@@ -269,12 +272,7 @@ export class SiteNotice extends React.Component {
 				<QueryActivePromotions />
 				{ siteRedirectNotice }
 				{ showJitms && (
-					<AsyncLoad
-						require="blocks/jitm"
-						messagePath={ messagePath }
```

Comment:
`messagePath` changes on each route change and causes a full re-render of the component due to prop changes. Moving it down into the component itself avoids this.